From b103c4a9e0c93f18576205eee7b8e0989aaf559d Mon Sep 17 00:00:00 2001
From: Pavel Moravec <pmoravec@redhat.com>
Date: Tue, 12 Nov 2019 19:12:45 +0100
Subject: [PATCH] [man] describe --allow-system-changes (#1850)

In #1435, --allow-system-changes option was added that is documented
in sosreport --help but not in manpages.

Resolves: #1850

Signed-off-by: Pavel Moravec <pmoravec@redhat.com>
Signed-off-by: Bryan Quigley <bryan.quigley@canonical.com>
---
 man/en/sosreport.1 | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/man/en/sosreport.1 b/man/en/sosreport.1
index 649cba0..a885d56 100644
--- a/man/en/sosreport.1
+++ b/man/en/sosreport.1
@@ -25,6 +25,7 @@ sosreport \- Collect and package diagnostic and support data
           [--log-size]\fR
           [--all-logs]\fR
           [--since YYYYMMDD[HHMMSS]]\fR
+          [--allow-system-changes]\fR
           [-z|--compression-type method]\fR
           [--encrypt-key KEY]\fR
           [--encrypt-pass PASS]\fR
@@ -160,6 +161,9 @@ increase the size of reports.
 Limits the collection to logs newer than this date.
 This also affects \--all-logs. Will pad with 0s if HHMMSS isn't specified.
 .TP
+.B \--allow-system-changes
+Run commands even if they can change the system (e.g. load kernel modules).
+.TP
 .B \-z, \--compression-type METHOD
 Override the default compression type specified by the active policy.
 .TP
-- 
1.8.3.1

